bundle-ml/src/main/scala/ml/combust/bundle/dsl/Bundle.scala

```diff
@@ -104,7 +105,8 @@ object Bundle {
     apply(BundleInfo(uid = UUID.randomUUID(),
       name = name,
       format = format,
-      version = Bundle.version), root)
+      version = Bundle.version,
+      LocalDateTime.now().toString), root)
```

Comment:
Yea they are both in the iso8601 format, and I have reflected the other two changes.